src/features/pokestop/PokestopPopup.jsx

```diff
@@ -129,7 +129,7 @@ export function PokestopPopup({
                       }}
                     >
                       <QuestConditions {...quest} />
-                      {quest.quest_shiny_probability && (
+                      {!!quest.quest_shiny_probability && (
```

Comment:
👀 What is this fixing? Was it not working before?

Reply:
See comments in Discord, weird symbols :)